cpp/src/Slice/Parser.h

```diff
@@ -390,13 +390,13 @@ namespace Slice
         /// Returns the mapped identifier that this element will use in the target language.
         [[nodiscard]] std::string mappedName() const;
 
```

Comment:
Bernard realized these doc-comments were actually flipped!
Unrelated to these changes, but good to fix.